schematic_api/api/openapi/api.yaml

```diff
@@ -101,8 +101,57 @@ paths:
         - Manifest Operations
   /manifest/download:
     get:
-      summary: Endpoint to download an existing manifest
-      description: Endpoint to download an existing manifest
+      summary: Endpoint to download an existing manifest by using a manifest id
+      description: Endpoint to download an existing manifest by using a manifest id
+      parameters:
+        - in: query
+          name: input_token
+          schema:
+            type: string
+            nullable: false
+          description: Token
+          example: Token
+          required: true
+        - in: query
+          name: manifest_id
+          schema:
+            type: string
+            nullable: true
+          description: manifest id
+          example: syn51078535
+          required: true
+        - in: query
+          name: new_manifest_name
+          schema:
+            type: string
+            nullable: true
+          description: Fill in if you want to change the filename of the downloaded manifest.
+          required: false
+        - in: query
+          name: as_json
+          schema:
+            type: boolean
+            default: True
+          description: if True return the manifest in JSON format. Default to True
+          required: false
+      operationId: schematic_api.api.routes.download_manifest
+      responses:
+        "200":
+          description: A manifest gets downloaded and local file path of the manifest gets returned.
+          content:
+            text/csv:
+              schema:
+                type: string
+        "403":
+          description: You don't have access to the requested resources. 
+        "500":
+          description: Please check schematic log
+      tags:
+        - Manifest Operations            
+  /dataset/manifest/download:
```

Comment:
Same thought here about API design

```
GET /dataset/{datasetId}/manifest
```

Reply:
This can also be a Jira ticket

Reply:
Thanks Tom. Thomas has mentioned similar things. Please see the jira ticket here: https://sagebionetworks.jira.com/browse/DCA-138